deploy/internal/statefulset-core.yaml

```diff
@@ -141,3 +141,23 @@ spec:
               valueFrom:
                 resourceFieldRef:
                   resource: limits.memory
+        - name: noobaa-log-processor
+          image: noobaa-core:s3-tests
```

Comment:
notice that this is just a place holder, and it has to be changed to the real image. you can use `NOOBAA_CORE_IMAGE` place holder as we do for core container

Reply:
Done